src/pages/stepsCalculator/stepsCalculator.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import {
+    IonButton,
+    IonCol,
+    IonContent,
+    IonHeader,
+    IonInput,
+    IonItem,
+    IonLabel,
+    IonNote,
+    IonRouterLink,
+    IonTitle,
+    IonToolbar,
+    IonGrid,
+    IonRow,
+} from '@ionic/react';
+//import ExploreContainer from '../../components/ExploreContainer';
+import './stepsCalculator.css';
+
+const StepsCalculator: React.FC = () => {
+    const [username, setUsername] = useState();
+    const [password, setPassword] = useState();
+    const [isTouched, setIsTouched] = useState(false);
+    const [isValid, setIsValid] = useState<boolean>();
+    var steps: number = 0;
+    var heightIn: number = 0; // height inches param
+    var heightFt: number = 0; // height feet param
+    var strideConverter: number = 0.413; // conversion factor for height in inches to average stride in inches
+    var heightCm: number = 0.0; // for our metric friends
+    var time: number;
+    var stepsPerMile: number = 2250;
+    var miles: number;
+    var metric: boolean = false;
+
+    function calculate(ev: Event) {
+        miles = Number((ev.target as HTMLInputElement).value);
+        if (miles <= 0) {
+            return 0;
+        }
+
+        steps = miles * stepsPerMile;
+    }
+
+    function getHeightFt(ev: Event) {
+        heightFt = Number((ev.target as HTMLInputElement).value);
+        if (heightFt <= 0) {
+            return 0;
+        }
+    }
+
+    function getHeightIn(ev: Event) {
+        heightIn = Number((ev.target as HTMLInputElement).value);
+        if (heightIn <= 0) {
+            return 0;
+        }
+    }
+
+    function getHeightcm(ev: Event) {
+        heightCm = Number((ev.target as HTMLInputElement).value);
+        if (heightCm <= 0) {
+            return 0;
+        }
+    }
+
+    function placeHolder() {
+        // return steps.toString();
```

Comment:
Do we still need this commented out code?

Reply:
Nope, will be removed